package.json

```diff
@@ -287,15 +287,15 @@
     "@metamask-institutional/types": "^1.1.0",
     "@metamask/abi-utils": "^2.0.2",
     "@metamask/accounts-controller": "^17.2.0",
-    "@metamask/address-book-controller": "^4.0.1",
-    "@metamask/announcement-controller": "^6.1.0",
+    "@metamask/address-book-controller": "^5.0.0",
+    "@metamask/announcement-controller": "^7.0.0",
     "@metamask/approval-controller": "^7.0.0",
     "@metamask/assets-controllers": "^36.0.0",
-    "@metamask/base-controller": "^5.0.1",
+    "@metamask/base-controller": "^6.0.2",
     "@metamask/bitcoin-wallet-snap": "^0.3.0",
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
-    "@metamask/controller-utils": "^10.0.0",
+    "@metamask/controller-utils": "^11.0.2",
```

Comment:
A number of these packages have breaking changes related to the removal of `eth_sign`. It seems that we may be blocked by this PR: https://github.com/MetaMask/metamask-extension/pull/24756

Reply:
Yes indeed it's the case I wasn't able to upgrade `signature-controller` for the same reason

Reply:
Got it, OK, well at least that PR seems close to being merged